docs/tutorials/networks_seq2seq_nmt.ipynb

```diff
@@ -5,34 +5,21 @@
     "colab": {
```

Comment:
<p>Mind keeping the output of this cell. I think it's a nice way for users to scroll to the bottom and see a working translation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>